airflow/models.py

```diff
@@ -1313,8 +1321,8 @@ def run(
         # not 0-indexed lists (i.e. Attempt 1 instead of
         # Attempt 0 for the first attempt).
         msg = "Starting attempt {attempt} of {total}".format(
```

Comment:
We can keep it for now for demo / debug purpose as it doesn't really affect users. But this definitely will be extra information as we add the attempt number tab.